_pages/dev/components/shared-components/split-view.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Split View Component
+feature:
+  - name: Split View Component
+    spa_version: 3.0
+    cx_version: n/a
+---
+
+{% capture version_note %}
+{{ site.version_note_part1 }} 3.0 {{ site.version_note_part2 }}
+{% endcapture %}
+
+{% include docs/feature_version.html content=version_note %}
+
+The split view component manages the presentation of hierarchical content on a page. A split view consists of a multi-column interface that shows two or more views in parallel. Split views are useful for navigating multiple levels of content hierarchy, such as traversing a list to view each item in the list.
+
+A good example of where Spartacus uses the split view component is in B2B Commerce Organization (also known as My Company). Entities, such as business units, budgets and cost centers, are presented as lists of items that can be opened and edited. The split view component offers a clear layout that allows you to go through the lists and edit them as needed.
```

Comment:
what about: 
The split view component offers a convenient UI that allows you to go through the lists and edit them as needed.

Reply:
👍